### PR TITLE
Custom payment renderers integration improvement

### PR DESCRIPTION
**Pull request by rajeev-k-tomy**
Instead of asynchronously loading the custom payment renderers, `customRenderers.js` will be populated in such a way that we can statically load that file in order to obtain all the custom payment renderers configured.

### CODE REVIEW

File: src/reactapp/src/paymentMethods/customRenderers.js [hunk 1]
```diff
@@ -1,21 +1 @@
...
-export default async function getCustomRenderers() {
-  const paymentMethodRenderers =
-    await paymentFetcher.fetchDataFromPaymentMethods('renderers.js', []);
-
-  return paymentMethodRenderers.reduce(
-    (renderers, item) => ({ ...renderers, ...item }),
-    {}
-  );
-}
+export default {};
```
Comment (wigman) [inline src/reactapp/src/paymentMethods/customRenderers.js:1]: @rajeev-k-tomy Maybe add a note "Contents of this file will be overwritten by  `src/reactapp/scripts/configurePaymentMethods.js` automatically".

(and add that as a note to the new content when it's generated maybe? "Do not edit this file directly, it's auto generated by `src/reactapp/scripts/configurePaymentMethods.js`")

**Review by wigman — APPROVED** [review]
Possible improvement in adding a comment to the generated file, otherwise looks good!

File: src/reactapp/src/paymentMethods/customRenderers.js (hunk 1 above)
Comment (rajeev-k-tomy) [reply src/reactapp/src/paymentMethods/customRenderers.js:1]: Really good suggestion. This can be incorporated.

Comment (rajeev-k-tomy) [reply src/reactapp/src/paymentMethods/customRenderers.js:12]: Updated as per your suggestion

Comment (rajeev-k-tomy) [reply src/reactapp/src/paymentMethods/customRenderers.js:12]: Let me know the comment is fine.

File: src/reactapp/scripts/configurePaymentMethods.js [hunk 2]
```diff
@@ -23,9 +23,18 @@ module.exports = (async () => {
     'payment method'
   );
 
-  let content = 'export default {};\n';
   if (paymentMethodList.length) {
-    content = paymentMethodList
+    let content = `/**
+ * 📢 📢 📢 This file is unlikely to be changed. Handle with care. 🙅🏽 🙅🏽 🙅🏽
```
Comment (wigman) [inline src/reactapp/scripts/configurePaymentMethods.js:28]: This file is autogenerated and may be overwritten. Handle with care.

File: src/reactapp/scripts/configurePaymentMethods.js [hunk 3]
```diff
@@ -23,9 +23,18 @@ module.exports = (async () => {
...
+ *
+ * When you have custom payment repo integration, then the content of this file
+ * will be altered during performing "npm install". So, in most of the cases, you
```
Comment (wigman) [inline src/reactapp/scripts/configurePaymentMethods.js:31]: will be overwritten during performing "npm install". So, in most of the cases, you

File: src/reactapp/src/paymentMethods/customRenderers.js (hunk 1 above)
Comment (wigman) [reply src/reactapp/src/paymentMethods/customRenderers.js:12]: Almost perfect :D

`unlikely to be changed` > I understand you're saying it's unlikely you need to change this file, but it reads as "it is not likely this file will ever change", but it is in fact the opposite. The file will change *very likely* :)

Comment (rajeev-k-tomy) [reply src/reactapp/src/paymentMethods/customRenderers.js:12]: Applied the changes.

File: src/reactapp/scripts/configurePaymentMethods.js (hunk 3 above)
Comment (rajeev-k-tomy) [reply src/reactapp/scripts/configurePaymentMethods.js]: Applied the changes suggested.

**Pull request merged**